dada2_tutorial_16S.R

```diff
@@ -103,10 +103,10 @@ knitr::opts_chunk$set(eval = TRUE,
 #'
 #+ package installation, eval = FALSE, include=TRUE
 install.packages("BiocManager")
-BiocManager::install("dada2", version = "3.8")
+BiocManager::install("dada2", version = "3.14")
```

Comment:
Maybe we should add a note stating that this (3.14) was the current version of Bioconductor at the time of the update, and that this will automatically update the most current version of dada2 (so people know where to check for potential incompatibilities). I just re-installed and dada2 version is 1.22 now.

Reply:
I like this idea!